wiki/Beatmap_Discussion/en.md

```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
 
 ### Watch/Unwatch
 
-This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on `modding watchlist` located near the top of the page.
+This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on [`modding watchlist` link](https://osu.ppy.sh/beatmapsets/watches) located near the top of the page.
 
 ### Hype Train
 
-Hype train is similar to the old star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
+Hype train is similar to the previous star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
 
 > Adding praise ❤ will raise this beatmap's hype, making it more visible for nomination and ranking!
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
 
+If you want to add Hype, it must be done in the General (All Difficulties) tab, not the General or the Timeline tabs. Players are required to play through a difficulty in the beatmapset in order to add Hype to a beatmap.
```

Comment:
I still dont understand this first sentence.

It must be done in the General tab, not the General tab.

Reply:
![image](https://user-images.githubusercontent.com/14264992/34413393-d8b317b6-eba8-11e7-8a6d-b39e1bd541be.png)

Reply:
oh

Reply:
I'll make some changes (probably add code) on this line and with the note above, since they are about the same thing just in a note and outside of a note.